samples/multimodality/xDiT/video-generation/aibrix_vke_staging_video_cogvideo_parallel.yaml

```diff
@@ -0,0 +1,134 @@
+apiVersion: apps/v1 
+kind: Deployment 
+metadata:
+  name: cogvideox-2b 
+  labels: 
+    model.aibrix.ai/name: cogvideox-2b 
+    model.aibrix.ai/port: "6000" 
+spec: 
+  replicas: 1 
+  strategy: 
+    rollingUpdate: 
+      maxSurge: 1 
+      maxUnavailable: 1 
+    type: RollingUpdate 
+  selector: 
+    matchLabels: 
+      model.aibrix.ai/name: cogvideox-2b 
+  template: 
+    metadata: 
+      labels: 
+        model.aibrix.ai/name: cogvideox-2b 
+        model.aibrix.ai/port: "6000" 
+    spec: 
+      initContainers: 
+        - command:             
+            - aibrix_download 
+            - --model-uri 
+            - tos://aibrix-artifact-testing/models/CogVideoX-2b/
+            - --local-dir 
+            - /models/ 
+          env: 
+            - name: DOWNLOADER_NUM_THREADS 
+              value: "16" 
+            - name: TOS_ACCESS_KEY 
+              valueFrom: 
+                secretKeyRef: 
+                  key: TOS_ACCESS_KEY 
+                  name: tos-credential 
+            - name: TOS_SECRET_KEY 
+              valueFrom: 
+                secretKeyRef: 
+                  key: TOS_SECRET_KEY 
+                  name: tos-credential 
+            - name: TOS_ENDPOINT 
+              value: https://tos-s3-cn-beijing.ivolces.com 
+            - name: TOS_REGION 
+              value: cn-beijing 
+            - name: LOG_LEVEL
+              value: DEBUG  
+          image: aibrix-container-registry-cn-beijing.cr.volces.com/aibrix/runtime:v0.4.1-mm1
```

Comment:
what's the `v0.4.1-mm1`? does not v0.4.1 work in this case? If not, could you share building instruction for v0.4.1-mm1?